docs/views/how-tos/updating-the-kit.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'layouts/layout.html' %}
+
+{% block pageTitle %}
+  Updating the kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
+        Updating the kit
+      </h1>
+
+      <div class="nhsuk-inset-text">
+        <p>
+          If you have made any changes outside the <strong>app</strong> folder, this process will destroy those changes.
+          We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside <strong>app</strong>, and add them back after updating.
+        </p>
+      </div>
+
+      <ol>
+        <li>
+          <a href="/docs/install/download-zip">Download the latest prototype kit</a>
+        </li>
+        <li class="nhsuk-u-margin-top-4">
+          In your prototype, delete everything apart from the <strong>app</strong> and <strong>.git</strong> folder.
+          <img class="app-img-guide nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-4" alt="Screenshot of the project folder, highlighting the folders and files that are to be deleted" src="/images/updating-the-kit/delete.png" />
+          <img class="app-img-guide nhsuk-u-margin-bottom-0" alt="Screenshot of the project folder with the deleted folders and files" src="/images/updating-the-kit/deleted.png" />
+        </li>
+        <li>
+          Copy everything from the latest kit to your prototype, apart from the <strong>app</strong> folder.
+          <img class="app-img-guide nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-4" alt="Screenshot of the latest prototype kit folder" src="/images/updating-the-kit/copy.png" />
+          <img class="app-img-guide nhsuk-u-margin-bottom-0" alt="Screenshot of the latest prototype kit folder" src="/images/updating-the-kit/paste.png" />
+        </li>
+        <li>
+          Copy the <strong>config.js</strong> file from the app folder in the latest kit, to the <strong>app</strong> folder in your prototype.
+          <img class="app-img-guide nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-4" alt="Screenshot of the config files in the latest prototype kit foldetr" src="/images/updating-the-kit/paste.png" />
+          <img class="app-img-guide nhsuk-u-margin-bottom-0" alt="Screenshot of the config files in the latest prototype kit foldetr" src="/images/updating-the-kit/paste-config.png" />
+          <div class="nhsuk-inset-text nhsuk-u-margin-top-0">
+            <p>If you've made any changes to the <strong>config.js</strong> file in your prototype then you'll need to re-enter them in the new version of the file e.g. the service name.</p>
+          </div>
+        </li>
+        <li>
+          Check <strong>\app\assets\sass\components</strong> in the latest kit for any new components. Copy the files over to your prototype.
```

Comment:
We use backslashes here `\` for folder paths but on the rest of the documentation we use forward slashes '/'.

Also does this need to be syntax highlighted with `<code class="app-code">` like we do elsewhere?

Reply:
Updated to "/"

Reply:
We're only using the syntax highlight for code and not for things like file name or file location.